component/storage/couchdb/store.go

```diff
@@ -333,16 +333,6 @@ func (c *StoreCouchDB) Iterator(startKey, endKey string) storage.StoreIterator {
 	return &couchDBResultsIterator{log: c.log, store: c, resultRows: resultRows}
 }
 
-// Query executes a query using the CouchDB _find endpoint.
-func (c *StoreCouchDB) Query(findQuery string) (storage.StoreIterator, error) {
-	resultRows, err := c.db.Find(context.Background(), findQuery)
-	if err != nil {
-		return nil, fmt.Errorf("failed to query CouchDB using the find endpoint: %w", err)
-	}
-
-	return &couchDBResultsIterator{log: c.log, store: c, resultRows: resultRows}, nil
-}
-
 type couchDBResultsIterator struct {
 	log        logger
```

Comment:
Remove logger as well.

Reply:
This one also needs to be removed https://github.com/hyperledger/aries-framework-go-ext/blob/main/component/storage/couchdb/store.go#L363

Reply:
Good catch - done. I also removed the WithLoggerOption function.